recipe/meta.yaml

```diff
@@ -116,15 +116,15 @@ outputs:
       host:
         - {{ pin_subpackage('_py-xgboost-mutex', exact=True) }}
         - {{ pin_subpackage('libxgboost', max_pin='x.x.x') }}
-        - libxgboost =*={{ string_prefix }}_h*_{{ PKG_BUILDNUM }}
+        - libxgboost * {{ string_prefix }}_h*_{{ PKG_BUILDNUM }}
```

Comment:
This is what is recommended by the linter. However it generates errors on CI. Likely due to conda-build issues (potentially fixed in 25.3)

For now this doesn't look ready for prime time. So am going to revert. Can revisit after the dust settles a bit

Refs: 
* https://github.com/conda-forge/conda-smithy/issues/2262
* https://github.com/conda/conda-build/pull/5600
* https://github.com/conda-forge/conda-build-feedstock/pull/249